types/dygraphs/index.d.ts

```diff
@@ -1347,13 +1609,32 @@ declare class Dygraph {
     static DASHED_LINE: number[];
     static DOT_DASH_LINE: number[];
 
+    /**
+     * Standard plotters.
+     * {@link https://dygraphs.com/jsdoc/symbols/Dygraph.html#.Plotters}
+     */
     static Plotters: {
-        errorPlotter: any;
-        linePlotter: any;
-        fillPlotter: any;
-    }
-}
+        errorPlotter(event: any): void;
+        linePlotter(event: any): void;
+        fillPlotter(event: any): void;
+    };
 
-declare module "dygraphs" {
```

Comment:
This bit is required to support ES imports. Please add it back.

Reply:
That's not required because the folder name is the same as the npm package name. Types will be published correctly.

https://github.com/Microsoft/dtslint/blob/master/docs/no-declare-current-package.md